src/components/App/App.tsx

```diff
@@ -118,9 +121,12 @@ const App = () => {
           >
             <PassportRedemption screen={ScreenName.Claim} />
           </Route>
-          <Route path="/:seller_id/dashboard/:secret_id">
+          <Route exact path="/:seller_id/dashboard/:secret_id">
             <MerchantVoucherDashboard />
           </Route>
+          <Route exact path="/:seller_id/dashboard/:secret_id/v2">
```

Comment:
could do a url param here as well but the new url was easier

Reply:
This is fine for now